### ISSUE TITLE
Add flag to CLI to generate text READMEs

### ISSUE THREAD

**Issue by olistic** [olistic/warriorjs#47]
Level READMEs in markdown format could be great if you're using some kind of markdown processor to read them. If you're not planning on doing so, having the markdown syntax laying around creates noise. Running `warriorjs --no-markdown` should take care of that, and output to `README.txt` instead of `README.md`.

**Comment by olistic** [comment]
Hello @justdvnsh, that'd be awesome. How are you planning to tackle this?

**Comment by justdvnsh** [comment]
So, i guess , after adding the ```-n``` or ```--no-markdown``` arg in the argument file ```parseArgs.js``` and then planning to write a simple md to txt parsing engine using regex. Because I dont find any md to txt parsing libraries out there.. Also , the txt files , would not be containing the text decorations and images.

What do you think ? Will this approach work ?

**Comment by olistic** [comment]
There are some libraries that strip markdown formatting characters from text you can try, like [remove-markdown](https://github.com/stiang/remove-markdown), but I don't know how well they'll fit our use case. What if you just create another `ejs` template, but for the `README.txt`?

**Comment by justdvnsh** [comment]
Yes , using an ejs template seems like a pretty good option . We can make the characters dynamic . But still, we would need to change all the readme files to add the dynamic content. 

So , I guess , instead of attaching readme files with the game , we can attach those ejs templates with the static  data, in which the dynamic data ( i.e. markdown formatting chars )  will be populated on the fly , after the user specifying wether he needs markdown or text .

How about that ? Sounds good ?

**Comment by olistic** [comment]
If I understood correctly, you're suggesting using the existing template but making the markdown formatting characters visible only when markdown is enabled? I thought of that but I think the template will get too convoluted. That's why I suggest having another template/set of templates called `README.txt.ejs`. This also allows us to make different formatting choices if we wanted in the text README, like this:

**Markdown:**
```markdown
# Spartacus - beginner

## Level 1

...
```

**Text:**
```txt
Spartacus - beginner
--------------------

Level 1

...
```

**Comment by justdvnsh** [comment]
Yes ,you understood that correctly. Yes , even I was wondering, that it will convolute the template . But I didn't really understand the solution you gave . If we will be using the README.txt.ejs template , then what will get added to it ( what will be the dynamic data ?

**Comment by olistic** [comment]
The existing template is called `README.md.ejs`, which includes all the markdown formatting characters and is used to generate the markdown README (`README.md`). We can add another template called `README.txt.ejs` without markdown and use that to generate the text README (`README.txt`) when the `--no-markdown` flag is passed. The data we pass to the template in the render function should be the same for both templates, the only different is the file format and layout.

**Comment by justdvnsh** [comment]
Okay yes . That sounds a pretty good solution.

**Comment by olistic** [comment]
Go for it! Thanks

**Comment by justdvnsh** [comment]
Sure thing !
